numpy/linalg/linalg.py

```diff
@@ -1382,7 +1382,7 @@ def cond(x, p=None):
 
     Parameters
     ----------
-    x : (M, N) array_like
+    x : (..., M, N) array_like
```

Comment:
Opened https://github.com/numpy/numpy/issues/6364, although I'm not confident that I've phrased that issue correctly.